core/cmd/ocr_keys_commands_test.go

```diff
@@ -22,29 +22,21 @@ func TestOCRKeyBundlePresenter_RenderTable(t *testing.T) {
 	t.Parallel()
 
 	var (
-		createdAt = time.Now()
-		updatedAt = time.Now().Add(time.Second)
-		deletedAt = time.Now().Add(2 * time.Second)
-		bundleID  = "7f993fb701b3410b1f6e8d4d93a7462754d24609b9b31a4fe64a0cb475a4d934"
-		buffer    = bytes.NewBufferString("")
-		r         = cmd.RendererTable{Writer: buffer}
+		bundleID = "b609c2e0e042cdb788de5234017a49103b489e6a9f94cb45ec3d34e1fe1a0f5f"
+		buffer   = bytes.NewBufferString("")
+		r        = cmd.RendererTable{Writer: buffer}
 	)
 
-	pk, err := ocrkey.NewKeyBundle()
-	require.NoError(t, err)
-	pkEncrypted, err := pk.Encrypt("p4SsW0rD1!@#_", utils.FastScryptParams)
+	key, err := ocrkey.NewV2()
```

Comment:
`MustNewV2XXXTestingOnly` ?

My reason for pushing on this is that, I believe generating real keys consumes entropy and if we do it a lot in tests on entropy-poor environments like CI it could be an actual bottleneck.

Reply:
ah I didn't realize it may function differently on CI. I replaced all possible key gen in tests with the default keys (where feasible, I left them in whenever import cycles were created). Including the one above. I can at least say it's an _improvement_ from the amount of keygen we had in tests before.